gammapy/maps/axes.py

```diff
@@ -2000,7 +2000,7 @@ def from_table(cls, table, format="gadf"):
                 except (KeyError, TypeError):
                     try:
                         axis = TimeMapAxis.from_table(table, format=format, idx=idx)
-                    except (KeyError, ValueError):
+                    except (KeyError, ValueError, IndexError):
```

Comment:
The `TimeMapAxis.from_table()` expects the table meta to have TIME_MIN and TIME_MAX instead of just TIME.
Thus, it tries to read `colnames[1]`, which does not exist for a node centered normal `MapAxis`. The addition of `IndexError` ensures that `Map.read` tries to read it as a normal `MapAxis`